src/main/resources/db/migration/V6.1.0.4__9356-add-rate-limiting.sql

```diff
@@ -0,0 +1 @@
+ALTER TABLE authenticateduser ADD COLUMN IF NOT EXISTS ratelimittier int DEFAULT 1;
```

Comment:
So superusers get tier 1 but it doesn't apply to them. I guess this is good for if they get demoted to regular users.